app/grandchallenge/challenges/forms.py

```diff
@@ -507,17 +507,18 @@ def __init__(self, *args, **kwargs):
 
     def clean_status(self):
         status = self.cleaned_data.get("status")
+
         if (
             status == self.instance.ChallengeRequestStatusChoices.ACCEPTED
             and Challenge.objects.filter(
                 short_name=self.instance.short_name
             ).exists()
         ):
             raise ValidationError(
-                f"There already is a challenge with short "
-                f"name: {self.instance.short_name}. Contact "
-                f"support@grand-challenge.org to accept this request.",
+                "There already is a challenge with short name. "
+                "Please contact support to accept this request.",
```

Comment:
This is in a form that only site admins use to accept challenge requests, they should know how to contact support.